src/KVNR.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/**
+ * @author    Davide Danna
+ * @copyright 2020 GAIA AG, Hamburg
+ * @package   KVNR-Validator
+ *
+ * Created using PhpStorm at 07.09.20 10:36
+ */
+
+namespace GaiaGroup;
+
+/**
+ *  KVNr utility tools class
+ */
+class KVNR
+{
+    /**
+     * Checks if given KVNR code is correct by validating its format and by verifying its checksum
+     *
+     * @param string $kvnr KVNR string to be validated
+     *
+     * @return bool true / false
+     */
+    public static function validate(string $kvnr): bool
+    {
+        // checks KVNR format using regex
+        if (preg_match("/^[A-Z][0-9]{9}$/", $kvnr)) {
+            // computes CRC digit using KVNR string trimmed out of the last digit
+            $kvnrCrc = self::computeCrc(substr($kvnr, 0, strlen($kvnr) - 1));
+
+            // compares computed CRC values with last digit in provided KVNR
+            return $kvnr[9] == $kvnrCrc;
+        }
+
+        return false;
+    }
+
+    /**
+     * returns the CRC digit for a given KVNR code. If provided string has not a valid format
```

Comment:
I think we should add an explanation of what CRC stands for, or remove this acronym altogether

Reply:
a more clear denomination can be 'check digit', what do you think about it? For a detailed explanation I linked the wiki page on the next line.

Reply:
That would be perfect 👍